e2e/src/helpers.ts

```diff
@@ -1,25 +1,26 @@
-import type { Page, Browser } from "@playwright/test";
-import { generateAuthenticationToken } from "./context";
+import { expect } from "@playwright/test";
+import type { Page, Browser, Locator } from "@playwright/test";
+import { findSessionId, generateAuthenticationToken } from "./context";
+import type { Context } from "./context";
+import type { GraphQLClient } from "graphql-request";
 
 // utility functions
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
 export function log(...args: any[]) {
-  process.env.DEBUG
-    ? console.log(...args)
-    : () => {
-      // silent
-    };
+  process.env.DEBUG_LOG && console.log(...args);
 }
 
 // a collection of useful playwright actions
+// these could evolve into fixtures: https://playwright.dev/docs/test-fixtures
```

Comment:
Helpful thank you - this is the bit of documentation I needed pointed towards when we were talking yesterday. I'll read up 👍